skietbaan_fe/src/components/Posts.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux';
+import { fetchPosts } from '../actions/postActions'
+
+class Posts extends Component {
+
+  // a constructor that maintanes the state of the component
+ /**  constructor(props ){
+    super(props);
+    this.state = {
+      posts: []
+    }
+  } */
+componentWillMount(){
+  this.props.fetchPosts();
+
```

Comment:
Whitespace - some unnecessary extra space in code that needs to be removed

Reply:
Sorted. Kindly review